src/mame/includes/centiped.h

```diff
@@ -69,6 +70,7 @@ class centiped_state : public driver_device
 	required_device<screen_device> m_screen;
 	required_device<palette_device> m_palette;
 	optional_device<ay8910_device> m_aysnd;
+	optional_memory_bank m_rombank;
```

Comment:
Please make a derived state class so you can make this a `required_memory_bank` and get the multipede-specific stuff out of here.

Reply:
Should I change all of `centiped_state`'s private members to protected when doing this, or are there some that should stay private?

Reply:
Well, if there are things that only need to be used by handlers in the base class, or handlers specific to systems in the base class, they can stay private.  Try not to expose everything if it doesn’t need to be exposed.